README.md

```diff
@@ -39,6 +39,20 @@ Start RedNotebook:
   * Linux/macOS: `python3 rednotebook/journal.py`
   * Windows: `py rednotebook/journal.py`
 
+## Run tests and pre-commit hooks
+
+Install:
+    * [tox](https://tox.wiki/en/latest/)
+    * [pre-commit](https://pre-commit.com/index.html)
+
+Run all tests:
+
+`tox`
+
+Run the pre-commit hooks:
+
+`pre-commit run`
+
```

Comment:
Let's separate this into two sections: the specific commands for setting up pre-commit and the commands for installing tox + running all tests.

Also, the tox tests should run the pre-commit tests, as in https://github.com/jugmac00/flask-reuploaded/blob/6496f8427a06be4a9f6a5699757ca7f9aba78ef6/tox.ini#L24-L26 (for this, install pre-commit inside tox).